pkg/move/cmd.go

```diff
@@ -180,3 +247,13 @@ func copyFile(fs afero.Fs, sourcePath string, destinationPath string) error {
 
 	return nil
 }
+
+func contains(slice []string, item string) bool {
+	for _, s := range slice {
+		if strings.Contains(s, item) {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
`slices.Contains` is better 😉

Reply:
done https://github.com/Dynatrace/dynatrace-bootstrapper/pull/4/commits/3bc29f55aae9afcb84fa1845556cbb335785cdf1